scan/src/components/validator/ProposedBlocksTable.re

```diff
@@ -0,0 +1,106 @@
+module Styles = {
+  open Css;
+
+  let vFlex = align => style([display(`flex), flexDirection(`row), alignItems(align)]);
+
+  let tableWrapper = style([padding2(~v=`px(20), ~h=`px(15))]);
+
+  let fixWidth = style([width(`px(230))]);
+
+  let icon = style([width(`px(80)), height(`px(80))]);
+  let iconWrapper =
+    style([
+      width(`percent(100.)),
+      display(`flex),
+      flexDirection(`column),
+      alignItems(`center),
+    ]);
+
+  let withWidth = w => style([width(`px(w))]);
+
+  let fillLeft = style([marginLeft(`auto)]);
+};
+
+[@react.component]
+let make = () => {
+  let blocksOpt = BlockHook.latest(~limit=5, ());
+  let blocks = blocksOpt->Belt.Option.getWithDefault([]);
+
+  let numProposedBlocks = 241;
+
+  <div className=Styles.tableWrapper>
+    <Row>
+      <HSpacing size={`px(25)} />
+      <Text value={numProposedBlocks |> string_of_int} weight=Text.Bold />
+      <HSpacing size={`px(5)} />
+      <Text value="Blocks" />
+    </Row>
+    <VSpacing size=Spacing.lg />
+    {blocks->Belt_List.length > 0
+       ? <>
+           <THead>
+             <Row>
+               <Col> <HSpacing size=Spacing.lg /> </Col>
+               <Col size=1.4>
+                 <Text block=true value="BLOCK" size=Text.Sm weight=Text.Bold color=Colors.gray6 />
+               </Col>
+               <Col size=2.15>
+                 <Text block=true value="AGE" size=Text.Sm weight=Text.Bold color=Colors.gray6 />
+               </Col>
+               <Col size=6.8>
+                 <Text
+                   block=true
+                   value="BLOCK HASH"
+                   size=Text.Sm
+                   weight=Text.Bold
+                   color=Colors.gray6
+                 />
+               </Col>
+               <Col size=1.5>
+                 <div className={Styles.vFlex(`flexEnd)}>
+                   <div className=Styles.fillLeft />
+                   <Text block=true value="TXN" size=Text.Sm weight=Text.Bold color=Colors.gray6 />
+                 </div>
+               </Col>
+               <Col> <HSpacing size=Spacing.lg /> </Col>
+             </Row>
+           </THead>
+           {blocks
+            ->Belt.List.map(({height, timestamp, hash, numTxs}) => {
+                <TBody key={hash |> Hash.toHex(~upper=true)}>
+                  <Row>
+                    <Col> <HSpacing size=Spacing.lg /> </Col>
+                    <Col size=1.5> <TypeID.Block id={ID.Block.ID(height)} /> </Col>
+                    <Col size=2.3> <TimeAgos time=timestamp code=true size=Text.Md /> </Col>
+                    <Col size=6.8>
+                      <div className={Styles.withWidth(500)}>
+                        <Text
```

Comment:
Design use font-weight 500 look bolder than this code

Reply:
But using 500 it look bolder that design choose what you want and reply this comment

Reply:
500 is look bolder than the design. So I think it's ok to stick with this.